lib/spack/spack/cmd/create.py

```diff
@@ -164,6 +168,10 @@ def install(self, spec, prefix):
         scons('prefix={0}'.format(prefix))
         scons('install')""",
 
+    'bazel': """\
+        # FIXME: Add logic to build and install here.
+        bazel('build')""",
```

Comment:
Huh, well that's kind of nice that it downloads dependencies for you, except when packaging it in a package manager. But `setup.py` does the same thing and downloads dependencies from PyPi if they aren't found. We just need to be careful about adding dependencies. I admit that I have little to no knowledge of Bazel, so I guess this is open for discussion.